README.md

```diff
@@ -0,0 +1,21 @@
+# Centos Developer VM Setup Script
+
+This script installs a few tools to get started with software development on Centos Server Virtual Machine.
+
+It installs
+* Golang
+* Gnome Desktop
+* XRDP
+* TigerVNC
+* VSCODE
+* Docker
+
+## Usage
+
+To use this script run this command
+```bash  
+git clone https://github.com/jritchieBAE/new-instance-setup.git && chmod +x ./new-instance-setup/setup.sh && sudo ./new-instance-setup/setup.sh
```

Comment:
Unconfirmed but I believe the chmod isn't required (I haven't needed to use it in testing)